2023.03/Dockerfile

```diff
@@ -31,14 +31,14 @@ ENV PATH $M2_HOME/bin:$PATH
 ENV JAVA_HOME=/usr/lib/jvm/java-17-openjdk-amd64
 ENV JDK_HOME=${JAVA_HOME}
 ENV JRE_HOME=${JDK_HOME}
-ENV MAVEN_VERSION=3.9.1
+ENV MAVEN_VERSION=3.9.0
```

Comment:
If this is the April release, is this correct?

Reply:
@JalexChen I had accidentally overwritten the march release by accidentally typing 3 instead of 4 when I ran the release script so I was reverting those changes